Relax restriction of arbitration ID uniqueness for SocketCAN (#785)
hardbyte/python-can: doc/interfaces/socketcan.rst

SocketCAN
=========

The `SocketCAN`_ documentation can be found in the Linux kernel docs at

> ``networking`` directory. Hereafter is quote from SocketCAN Linux document::

Reviewer: ```suggestion
``networking`` directory. Quoting from the SocketCAN Linux documentation::
```
